### ISSUE TITLE
The issue with hostname within the endpoints

### ISSUE THREAD

**Issue by Eugene015** [go-spatial/tegola#1025]
Hi guys,
I installed the current version of tegola on my Ubuntu 24.04.1 LTS. But I have issue with my tegola server. It does not see hostname. I mean it is missing in network responses, however, I wrote that in tegola config file. What could it be? 

This is my tegola config:
[webserver]
port = ":9000"
hostname = "localhost" 

[[providers]]
name = "my_geodb"
type = "postgis"
uri = "postgresql://postgres:123456@localhost:5433/my_geodb?sslmode=disable"

  [[providers.layers]]
  name            = "kyiv"
  table           = "public.kyiv"
  geometry_field  = "wkb_geometry"
  geometry_type   = "MULTIPOLYGON"
  srid            = 4326
  id_field        = "ogc_fid"

[[maps]]
name = "kyiv_map"

  [[maps.layers]]
  name = "kyiv_layer"
  provider_layer = "my_geodb.kyiv"

And this is the response example on http://localhost:9000/maps/kyiv_map/style.json:
{
    "version": 8,
    "name": "kyiv_map",
    "center": [
        0,
        0
    ],
    "sources": {
        "kyiv_map": {
            "type": "vector",
            "url": "http:///capabilities/kyiv_map.json"
        }
    },
    "layers": [
        {
            "id": "kyiv_layer",
            "source": "kyiv_map",
            "source-layer": "kyiv_layer",
            "type": "fill",
            "layout": {
                "visibility": "visible"
            },
            "paint": {
                "fill-color": "rgba(45,175,193,0.1)",
                "fill-outline-color": "#2dafc1"
            }
        }
    ]
}

**Comment by ARolek** [comment]
@Eugene015 hey there. This was surfaced in another issue: https://github.com/go-spatial/tegola/issues/1022. I have fixed it in this PR: https://github.com/go-spatial/tegola/pull/1023. I have not yet cut the patch release as I'm waiting on the following PR to also land: https://github.com/go-spatial/tegola/pull/1024. In the meantime you can test the fix the following ways:

* Docker: pull down the `edge` tag: https://hub.docker.com/layers/gospatial/tegola/edge/images/sha256-8f679a088a2f4732a9326f01932b7ae3593789e531fad8f799156585beb0af6c
* Binaries: there are build artifacts you can download from the Github Action here: https://github.com/go-spatial/tegola/actions/runs/12472282702
* Compile from source: You can pull down `master` and compile from source using these instructions: https://github.com/go-spatial/tegola?tab=readme-ov-file#building-from-source.

Please give this a test and report back if your issue is fixed. We will cut a new patch release soon!

**Comment by ARolek** [comment]
Closing issue as this is fixed in `master` and will be part of this week's release.

**Comment by Eugene015** [comment]
Thanks
